18-minimal/README.md

```diff
@@ -313,6 +316,47 @@ Below is an example _package.json_ file with the _main_ attribute and _start_ sc
 #### Note:
 `oc rsync` is only available in versions 3.1+ of OpenShift.
 
+## init-wrapper
```

Comment:
I think we should also mention the INIT_WRAPPER env variables in the `Environment variables for Source-to-Image` above even if they only refer to this later section.